core/templates/pages/exploration-player-page/layout-directives/learner-local-nav.component.html

```diff
@@ -3,7 +3,7 @@
 </ng-template>
 <ul class="nav navbar-nav oppia-navbar-nav float-right mr-0">
   <li class="nav-item">
-    <a href="" class="nav-link" ngbTooltip="Share"
+    <a href="javascript:void(0)" class="nav-link" ngbTooltip="Share"
```

Comment:
Thanks for the fix! Should we just replace the anchor tag here with a more semantically relevant one but keep the styling?

Reply:
That makes sense, would `span` be a better choice here?

Reply:
Yup, sounds good! BTW, in the future, if you would like a reviewer to take a pass on the PR or would like them to take a look at something, please feel free to leave a comment like the following: "{{Question/comment}} @{{reviewer_username}} PTAL". Oppiabot will help assign that person for you (as mentioned in "PR Pointers" in the description.) Thanks!